search-app/docs/api.md

```diff
@@ -241,7 +240,7 @@ The CORS headers are supported on search endpoints. Check [CORS Documentation](h
  * `scroll` - A boolean flag (true/false) that allows all results to be retrieved efficiently. `page_size` is supported with `scroll` while `page_num` and `offset` are not. If `scroll` is `true` then the first call of a scroll session sets the page size; `page_size` is ignored on subsequent calls.
  * `sort_key` - Indicates one or more fields to sort on. Described below
  * `pretty` - return formatted results if set to true
- * `token` - specifies a user/guest token from EDL or Launchpad to use to authenticate yourself. EDL tokens can be specified as a header with the `Authorization: Bearer` prefix. Launchpad tokens can be specified as the header with `Authorization:`. 
+ * `token` - specifies a user/guest token from [EDL](https://urs.earthdata.nasa.gov/) or [Launchpad](https://launchpad.nasa.gov) for use as authentication however this practice should be considered deprecated in favor of using the standard [Authorization header](#header).
```

Comment:
```suggestion
 * `token` - specifies a user/guest token from [EDL](https://urs.earthdata.nasa.gov/) or [Launchpad](https://launchpad.nasa.gov) for use as authentication. However, this practice should be considered deprecated in favor of using the standard [Authorization header](#header).
```

Reply:
I don't think this description should try to include links to URS or Launchpad. We don't do this in other token references in in the docs and, importantly, these links aren't actually to the respective API endpoints for managing tokens.

Reply:
And I think we need to run the "deprecation" language by Joe and Ryan. We can't unilaterally make that assertion. The query parameter exists and is fully supported, further deletion of the parameter (and support thereof) has been raised and rejected. I apologize @jceaser for giving the wrong impression